src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/MigrationTests.java

```diff
@@ -137,6 +136,29 @@ public void testSecurityValidateWithInvalidConfig() throws Exception {
 
     }
 
+    @Test
+    public void testSecurityMigrateWithEmptyPassword() throws Exception{
+        final Settings settings = Settings.builder().put(SSLConfigConstants.OPENDISTRO_SECURITY_SSL_HTTP_CLIENTAUTH_MODE, "REQUIRE")
+                .put("opendistro_security.ssl.http.enabled", true)
+                .put("opendistro_security.ssl.http.keystore_filepath", FileHelper.getAbsoluteFilePathFromClassPath("migration/node-0-keystore.jks"))
+                .put("opendistro_security.ssl.http.truststore_filepath", FileHelper.getAbsoluteFilePathFromClassPath("migration/truststore.jks"))
+                .put(ConfigConstants.OPENDISTRO_SECURITY_UNSUPPORTED_ACCEPT_INVALID_CONFIG, true)
+                .build();
+        setup(Settings.EMPTY, new DynamicSecurityConfig().setSecurityInternalUsers("internal_users2.yml").setLegacy(), settings, true);
+        final RestHelper rh = restHelper(); //ssl resthelper
+
+        rh.enableHTTPClientSSL = true;
+        rh.trustHTTPServerCertificate = true;
+        rh.sendAdminCertificate = true;
+        rh.keystore = "kirk-keystore.jks";
+        
+        String body = "{\"internalusers\":\"eyJsb2dzdGFzaCI6eyJoYXNoIjoiIiwicm9sZXMiOlsibG9nc3Rhc2giXX0sIlN0ZXBoZW5fMTIzIjp7Imhhc2giOiIiLCAicGFzc3dvcmQiOiIifSwic25hcHNob3RyZXN0b3JlIjp7Imhhc2giOiIiLCJyb2xlcyI6WyJzbmFwc2hvdHJlc3RvcmUiXX0sImFkbWluIjp7ImF0dHJpYnV0ZXMiOnsiYXR0cmlidXRlMSI6InZhbHVlMSIsImF0dHJpYnV0ZTMiOiJ2YWx1ZTMiLCJhdHRyaWJ1dGUyIjoidmFsdWUyIn0sInJlYWRvbmx5IjoidHJ1ZSIsImhhc2giOiIiLCJyb2xlcyI6WyJhZG1pbiJdfSwia2liYW5hc2VydmVyIjp7InJlYWRvbmx5IjoidHJ1ZSIsImhhc2giOiIifSwia2liYW5hcm8iOnsiaGFzaCI6IiIsInJvbGVzIjpbImtpYmFuYXVzZXIiLCJyZWFkYWxsIl19LCJyZWFkYWxsIjp7Imhhc2giOiIiLCJyb2xlcyI6WyJyZWFkYWxsIl19fQoK\"}";
```

Comment:
Is it necessary to hardcode base64 encoded string? Can it be programmatically encoded for readability?

Reply:
Done.